.github/workflows/pre-release.yml

```diff
@@ -346,9 +346,28 @@ jobs:
         uses: jaxxstorm/action-install-gh-release@v1.10.0
         with:
           repo: pulumi/pulumictl
-      - name: Add SDK version tag
-        run: git tag "sdk/v$PROVIDER_VERSION" && git push origin
-          "sdk/v$PROVIDER_VERSION"
+      - name: Download Go SDK
+        uses: actions/download-artifact@v3
```

Comment:
Most of our actions outside of this repo (including ci-mgmt) use v4. Could we fail forward into using v4 everywhere?

Reply:
I agree that upgrading to v4 should be the long term goal. However, our Go SDK is unusable currently which is why I opted to use v3 to avoid any other unexpected failures during the release workflow and unbork the Go SDK ASAP.